code/game/objects/structures/tables_racks.dm

```diff
@@ -277,6 +277,7 @@
 	icon = 'icons/obj/smooth_structures/rollingtable.dmi'
 	icon_state = "rollingtable"
 	var/list/attached_items = list()
+	///Used to get the last turf this table was on so we can add the TRAIT_PROTECT_FOOD trait to it
```

Comment:
?

Reply:
Right, my bad, that was from when tables put traits on floors. I removed the code but forgot to remove the comment